deploy/crds/apps.open-cluster-management.io_multiclusterapplicationsetreports.yaml

```
@@ -21,8 +22,12 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: MulticlusterApplicationSetReport is the Schema for the MulticlusterApplicationSetReport
-          API.
+        description: MulticlusterApplicationSetReport provides a report of the status
+          of an application from all the managed clusters where the application is
+          deployed on. It provides a summary of the number of clusters in the various
```

let's not end on a preposition. 

```suggestion
          deployed. It provides a summary of the number of clusters in the various
```